Assets/HoloToolkit/Utilities/Scripts/SceneLauncherChecker.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public class SceneLauncherChecker : MonoBehaviour
```

Comment:
Because `SceneLauncher` is a singleton, and I think there's some initialization that happens.  I'll look into how we can make this baked into the scene.